mypy/checkmember.py

```diff
@@ -96,155 +96,179 @@ def analyze_member_access(name: str,
     return _analyze_member_access(name, typ, mx, override_info)
 
 
+# The several functions that follow implement analyze_member_access for various
+# types and aren't documented individually.
```

Comment:
I would actually move this comment after `_analyze_member_acces` itself, e.g. to line 135.